### DISCUSSION TITLE
Shopee.co.id - Banner Download Aplikasi

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1010] [Personal Filter]
Awal buka langsung disuguhi pop ups
Ini setting belum masuk akun

<details>
  <summary>Screenshot</summary>
  
  ![Screenshot_20211022-100141398.jpg](https://user-images.githubusercontent.com/19517680/138386562-825c7366-fd32-4fc9-ba56-328cc517fe37.jpg)
</details>

**Comment by realodix** [comment]
Sorry Gan @3xploiton3, Saya pribadi menambahkannya ke personal filter dan issue Saya pindahkan ke discussions. 

```
shopee.co.id##.shopee-popup
shopee.co.id##.home-popup
shopee.co.id#$#.shopee-no-scroll { overflow: auto!important; }
shopee.co.id#$#body { overflow: auto!important; }
```

Agan bisa menambahkannya ke personal filter Agan atau langsung melaporkannya ke filter yang memang khusus menangani annoyance seperti [AdGuard Annoyances](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter/sections) atau [Fanboy’s Annoyance](https://github.com/easylist/easylist/tree/master/fanboy-addon).

Alasannya karena dalam konteks ini self promotions (dengan berbagai bentuknya di berbagai website) dalam survei banyak juga user yang tidak masalah karena self-promotion walaupun cukup mengganggu tapi juga mengandung manfaat untuk mereka. Karena filter annoyace jadi satu dengan filter utama, tidak ada pilihan lain untuk mereka selain menonaktifkan AdBlockID secara keseluruhan.

Annoyance yang Saya tangani yang benar-benar tidak ada manfaatnya di sisi user:
- Mouse limitation (gak bisa klik kanan, gak bisa select text, dll)
- Key limitation (gak bisa print screen)
- Fake button
- Popup & Overlay yang disalahgunakan untuk menampilkan iklan
- Safelink yang disalahgunakan untuk menampilkan iklan

Jika ada hal lain untuk personal filter, Agan bisa diskusikan di halaman [discussions](https://github.com/realodix/AdBlockID/discussions).

**Reply by 3xploiton3** [reply]
Klo yg saya maksud itu tampilan mobile,
Hampir semua orang rata2 menggunakan aplikasi shopee official (apk/ios) dibanding web mobile krn saya merasa terganggu dgn iklan yg tdk relevan dan pop up yg sering ganggu, padahal isi pop up tsb ada di banner web,

Klo saya pakai filter pribadi terlalu ribet, device yg saya gunakan banyak, 

Klo web desktop tdk masalah, saya kasih ke filter pribadi atau tdk, krn tdk terlalu menganggu

**Comment by 3xploiton3** [comment]
![Screenshot_20211217-161658019](https://user-images.githubusercontent.com/19517680/146520337-20e2cc1c-606e-4924-b9aa-b1c182ed49f7.jpg)

https://shopee.co.id/search?keyword=stainless

Filter sudah saya masukkan masih ada pop up

**Reply by realodix** [reply]
Saya coba mereplikasinya di PC (Edge dan Firefox), tapi sayanya gak muncul popup

**Reply by 3xploiton3** [reply]
> Saya coba mereplikasinya di PC (Edge dan Firefox), tapi sayanya gak muncul popup

maaf itu tampilan mobile

**Reply by realodix** [reply]
`shopee.co.id###modal:has(button:has-text(Download Sekarang))`

Untuk AdGuard menurut dokumentasi sih bisa pakai syntax uBO, tapi kalau gak bisa tinggal dirubah saja jadi
`shopee.co.id#?##modal:has(button:has-text(Download Sekarang))`

**Comment by 3xploiton3** [comment]
![Screenshot_20211231-190624130](https://user-images.githubusercontent.com/19517680/147822692-426a0f1a-a4a8-43b9-9163-4dc337224c95.jpg)

![Screenshot_20211231-191300981](https://user-images.githubusercontent.com/19517680/147822756-d0e4fad3-2a57-4efc-a210-4430303adedd.jpg)

https://shopee.co.id/search?keyword=timbangan

Tampilan tersebut pakai mobile, kiwi browser

Bisa minta tolong dibuatkan filternya @realodix, rule diatas sudah tidak bekerja,

Terimakasih

**Reply by realodix** [reply]
Item yang ada tulisan "AD" sudah Saya commit pakai syntax biasa di https://github.com/realodix/AdBlockID/issues/1094. Harusnya sih cocok untuk AdGuard, tinggal nunggu automatic converter mereka.
